docs/developers/contributing.rst

```diff
@@ -49,13 +53,298 @@ tests, then stop and consider the following strategies instead:
+required, and even the size estimation for the task. You can find tasks of all
+sizes in the academy issues list.
+
+From small tasks, such as replacing outdated terminology, checking for broken
+links, testing a tutorial or ensuring adherence to the
+`Canonical documentation style guide`_\ ; to medium-sized  tasks like,
+converting documentation from one format to another, or migrating the contents
+of a blog post into the official documentation; to more ambitious tasks, such
+as adding a new *How-to* guide, restructuring a group of documents, or
+developing new tests and automations.
+
+Completing and closing tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a task has been completed to your satisfaction, we’ll ask the contributor
+whether they would prefer to merge their work into your project themselves,
+or leave this to the project.
+
+Recognition
+^^^^^^^^^^^
+
+After successfully completing a task, we’ll give credit to the contributor
+and share their success in our forums, on the pages themselves, and in our
+news updates and release notes.
+
+Guidance for writing
+~~~~~~~~~~~~~~~~~~~~
+
+Consistency of writing style in documentation is vital for a good user
+experience. To accommodate our audience with a huge variation in experience,
+we:
+
+* write with our target audience in mind
+* write inclusively and assume very little prior knowledge of the reader
+* link or explain phrases, acronyms and concepts that may be unfamiliar, and if
+  unsure, err on the side of caution
+* adhere to the style guide
+
+Language
+^^^^^^^^
+
+MicroOVN documentation currently uses British (GB) English. However, Canonical
+recently switched to US English. It is our aim to switch to the US English as
+well. Until we completely switch over, the contributions should continue to
+use British English.
+
+There are many small differences between UK and US English, but for the most
+part, it comes down to spelling. Some common differences are:
+
+* the ``ize`` suffix in preference to ``ise`` (e.g. ``capitalize`` and
+  ``capitalise``)
+* *our* instead of *or* (as in ``color`` and ``colour``)
+* licence as both a verb and noun
+* ``catalog`` and ``catalogue``
+* dates take the format 1 January 2013, 1-2 January 2025 and 1 January - 2
+  February 2025
+
+We use an automated spelling checker that sometimes throws errors about terms
+we would like it to ignore:
+
+* If it complains about a file name or a command, enclose the word in double
+  backticks (``) to render it as inline code.
+* If the word is a valid acronym or a well-known technical term (that should
+  not be rendered as code), add it to the spelling exception list,
+  ``docs/.custom_wordlist.txt`` (terms should be added in alphabetical order).
+
+Both methods are valid, depending on whether you want the term to be rendered
+as normal font, or as inline code (monospaced).
+
+Acronyms
+^^^^^^^^
+
+Acronyms should always be capitalised.
```

Comment:
[* nit *] capitalised => capitalized (since we just said that we're moving to U.S. English.